tools/go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/submariner-io/shipyard/tools
 
-go 1.20
+go 1.21
+
+toolchain go1.21.3
```

Comment:
What is this? Haven't seen it before...

Reply:
Yeah I'm not sure, I see https://go.dev/ref/mod#go-mod-file-toolchain